src/sentry/static/sentry/app/routes.jsx

```diff
@@ -82,6 +82,7 @@ import ProjectReleaseTracking from './views/projectReleaseTracking';
 import ProjectReleases from './views/projectReleases';
 import ProjectSavedSearches from './views/projectSavedSearches';
 import ProjectSettings from './views/projectSettings';
+import ProjectTeams from './views/settings/project/projectTeams';
```

Comment:
Let's remove this and lazy load the view, see the "ApplicationDetails" route for an example.